qiita_db/processing_job.py

```diff
@@ -1031,14 +1031,36 @@ def processing_job_worflow(self):
             The processing job workflow the job
         """
         with qdb.sql_connection.TRN:
-            sql = """SELECT processing_job_workflow_id
-                     FROM qiita.processing_job_workflow_root
-                     WHERE processing_job_id = %s"""
+            # Retrieve the workflow root jobs
+            sql = "SELECT * FROM qiita.get_processing_workflow_roots(%s)"
```

Comment:
could you add what you are "SELECTing"? Same concern than before.

Reply:
Done